python-stubs/dolphin/__init__.pyi

```diff
@@ -1 +1 @@
-from . import controller, event, gui, memory, savestate
+from . import controller as controller, event as event, gui as gui, memory as memory, savestate as savestate
```

Comment:
In the latest release, `dolphin` isn't a module, I assume it is now a module in master? Otherwise this `__init__` would be wrong.